src/OpenTelemetry.AutoInstrumentation/Instrumentation.cs

```diff
@@ -113,9 +114,48 @@ private static void OnExit(object sender, EventArgs e)
             return;
         }
 
-        _tracerProvider.Dispose();
+        try
+        {
+            _tracerProvider.Dispose();
```

Comment:
Just curious: the try-catch here is more to log some info in the unlikely case of an exception. I mean you didn't actually found a case in which the `_traceProvider.Dispose()` is throwing, right?

Reply:
Correct. I didn't look to see if there's an existing case where that call could throw an exception. I put it in place to be extra defensive. It may not be necessary, because the process is terminating anyways, but I'd rather not take the chance of having an exception from our code potentially hide what happened in the application. Taking on #321 will introduce code that will have more potential for triggering an exception. Similarly, I've seen logging systems lead to exceptions too, or sometimes it's the interpolated string that leads to the exception.